modules/components/AsyncConnect.js

```diff
@@ -56,7 +56,25 @@ export default class AsyncConnect extends Component {
   }
 
   isLoaded() {
-    return this.context.store.getState().reduxAsyncConnect.loaded;
+    const componentKeys = this.props.components.reduce((componentKeysMemo, component) => {
+      if (component && component.reduxAsyncConnect && component.reduxAsyncConnect.length) {
+        const asyncConnectComponentKeys = component.reduxAsyncConnect
+          .filter(asyncConnectComponent => !!asyncConnectComponent.key)
+          .map(asyncConnectComponent => asyncConnectComponent.key);
```

Comment:
Could also be written as

.reduce((keysMemo, asyncConnectComponent) => (
  asyncConnectComponent.key ? [...keysMemo, asyncConnectComponent.key] : keysMemo
), [])
instead of .filter().map()

Whichever you think is cleaner.